MiniAn/minian_main.py

```diff
@@ -661,9 +661,9 @@ def save_minian_to_doric(
     print(mn_defs.Messages.GEN_ROI_NAMES, flush = True)
     names = []
     usernames = []
-    for i in range(len(C)):
-        names.append("ROI"+str(i+1).zfill(4))
-        usernames.append("ROI {0}".format(i+1))
+    for i in A.coords["unit_id"].values:
+        names.append("ROI"+str(i).zfill(4))
```

Comment:
names and usernames are used only when saving spikes, so I would move it there

Reply:
Somehow it doesn't feel right that we're generating the `names` and `usernames` twice: here and in the `save_roi_signals`

Reply:
I think it would be better to either to do it only there or give the   `roi_ids` to   `utils.save_signals(`  like in `utils.save_roi_signals(`

But  as a quick fix it is fine I think for nom. But if we change the functions we have to also change in caiman :(